packages/plugin-lighthouse/src/lib/utils.ts

```diff
@@ -68,3 +75,89 @@ export function validateOnlyAudits(
   }
   return true;
 }
+
+type UnsupportedDetail =
+  | Details.CriticalRequestChain
+  | Details.List
+  | Details.TreemapData
+  | Details.Screenshot
+  | Details.Filmstrip
+  | Details.DebugData;
+type UnsupportedDetailTypes = UnsupportedDetail['type'];
+
+export function toAuditOutputs(
+  lhrAudits: Result[],
+): AuditOutputs {
+  return Object.values(lhrAudits).map(
+    ({
+      id: slug,
+      score,
+      numericValue: value = 0, // not every audit has a numericValue
+      details,
+      displayValue,
+    }: Result) => {
+      const auditOutput: AuditOutput = {
+        slug,
+        score: score ?? 1, // score can be null
+        value,
+        displayValue,
+      };
+
+      if (details != null) {
+        const type = details.type;
+        switch (type) {
+          case 'opportunity':
+            return {
+              ...auditOutput,
+              details: opportunityToDetails(details),
+            };
+          case 'table':
+            return {
+              ...auditOutput,
+              details: tableToDetails(details),
+            };
+          default:
+            const unsupportedType: UnsupportedDetailTypes = type;
+            return {
+              ...auditOutput,
+              details: {
+                issues: [
+                  {
+                    message: `Parsing details from type ${unsupportedType} is not implemented.`,
+                    severity: 'info',
+                  },
+                ],
+              },
+            };
```

Comment:
I don't think this is a good way of dealing with unsupported features. I'd log a warning to the console instead of creating issues in the report.

Reply:
Returned the audit output without details and added a `console.info`.